next.config.js

```diff
@@ -1,6 +1,9 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  experimental: {
+    newNextLinkBehavior: false,
```

Comment:
I decided not to deal with fixing our usages of `<Link` in this PR, but I probably should.